### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by neutr0ncodes**
added some comment's in the backend and deleted the repeated code.

### CODE REVIEW

File: server/src/routes.ts [hunk 1]
```diff
@@ -8,36 +8,44 @@ type temp = {
   name: string;
 };
 
+//Check's the server is working fine or not
+
 function routes(app: Express) {
   app.get("/healthcheck", (req: Request, res: Response) => res.sendStatus(200));
 
+
```
Comment (adityadafe) [inline server/src/routes.ts:16]: remove this extra line

**Review by adityadafe — APPROVED** [review]
Make following changes

File: server/src/routes.ts (hunk 1 above)
Comment (neutr0ncodes) [reply server/src/routes.ts:16]: ye buddha mere bicchem me bahut bolta hai

File: server/src/routes.ts [hunk 2]
```diff
@@ -8,36 +8,44 @@ type temp = {
...
     }
     res.status(404).send("Enter valid URL!");
   });
-  app.post("/api/add", async (req: Request, res: Response) => {
-    const { email, name }: temp = req.body;
-    const newObj =
-      await SQL`insert into users(name ,email) values (${name} , ${email}) RETURNING *;`;
-    res.json(`User added with ID: ${newObj[0].id}`);
-  });
+
```
Comment (adityadafe) [inline server/src/routes.ts:44]: do something like - 
```ts
originalUrl ? res.redirect(originalUrl) : res.status(404).send("Enter valid URL!");
```

Comment (neutr0ncodes) [reply server/src/routes.ts:44]: done!

**Comment by adityadafe** [comment]
LGTM :rocket:

**Pull request merged**